modules/account-lib/.mocharc.js

```diff
@@ -2,7 +2,7 @@
 
 module.exports = {
   require: 'ts-node/register',
-  timeout: '20000',
```

Comment:
I tried to speed things up wherever possible by using async randomness, but turns out generating 3072 bit prime numbers takes time.